Tests/Invoke-Tests.ps1

```diff
@@ -0,0 +1,54 @@
+<#
+.SYNOPSIS
+Runner to invoke Acceptance, Quality and / or Unit tests
+
+.DESCRIPTION
+Test wrapper that invokes
+
+.PARAMETER TestType
+[Optional] The type of test that will be executed. The parameter value can be either All (default), Acceptance, Quality or Unit
+
+.EXAMPLE
+Invoke-AcceptanceTests.ps1
+
+.EXAMPLE
+Invoke-AcceptanceTests.ps1 -TestType Unit
+
+#>
+
+[CmdletBinding()]
+Param (
+    [Parameter(Mandatory = $false)]
+    [ValidateSet("All", "Acceptance", "Quality", "Unit")]
+    [String] $TestType = "All"
+)
+
+$TestParameters = @{
```

Comment:
camelCase local variable?

Reply:
I'll go through all of the scripts so far and ensure I enforce the new naming conventions.

Reply:
I've updated Invoke-Tests.ps1 to apply naming conventions.